apps/emqx_bridge/src/emqx_bridge_v2.erl

```diff
@@ -731,6 +736,14 @@ bridge_v2_type_to_connector_type(kafka_producer) ->
 bridge_v2_type_to_connector_type(azure_event_hub) ->
     azure_event_hub.
 
+%%====================================================================
+%% Data backup API
+%%====================================================================
+
+import_config(RawConf) ->
+    %% bridges v2 structure
+    emqx_bridge:import_config(RawConf, <<"bridges_v2">>, ?ROOT_KEY, config_key_path()).
```

Comment:
we need these callbacks to properly detect and pretty-print changes, as each part of the config can have its own structure. That's why can't simply call `emqx_conf:update/3` in `emqx_mgmt_data_backup` module...